README.md

```diff
@@ -134,6 +134,7 @@ This provider exposes quite a few provider-specific configuration options:
 * `subnet_id` - The subnet to boot the instance into, for VPC.
 * `associate_public_ip` - If true, will associate a public IP address to an instance in a VPC.
 * `tags` - A hash of tags to set on the machine.
+* `package_tags` - A hash of tags to set on the ami generated during the package operation.
```

Comment:
Good question. Because AMI is such a pervasive term, I wanted to avoid using it and risking confusion. We already use an AMI to start the original vagrant instance and somehow specifying that those were the tags for the "package" rather than "an AMI" seemed to be more sensible.